recipes/findspark/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: findspark
+  version: "1.0.0"
+
+source:
+  fn: findspark-1.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/b6/0b/42d86a026eb78a8d7660077fa65515ec529dce1ae85395898c977d0882f3/findspark-1.0.0.tar.gz
```

Comment:
Please change to the more "human readble" URL below :wink: 

https://pypi.io/packages/source/f/findspark/findspark-{{ version }}.tar.gz